layouts/shortcodes/sign-language-section.html

```diff
@@ -0,0 +1,22 @@
+{{/*
+
+  Usage: {{% sign-language-section VIDEO_ID %}}
+
+  */ -}}
+
+{{ $youtube := "" -}}
+{{ $videoID := .Get 0 -}}
+{{ if $videoID -}}
+  {{ $youtube = printf "{{< youtube id=%q >}}" (printf "%s?mute=1" $videoID) }}
+{{ else -}}
+  {{ $page := (and .Page.File .Page.File.Path) | default .Page.Title -}}
+  {{ warnf "%s: Shortcode 'sign-language-section' requires a video ID as argument" $page -}}
+{{ end -}}
+
+## {{ i18n "sign_language_header" }}
+
+<div class="sign-language__video">
+{{ $.Page.RenderString $youtube }}
+</div>
+
+{{ i18n "sign_language_info" }}
```

Comment:
```suggestion
{{ i18n "sign_language_info" }}

```

Reply:
@chalin 
Agreed with
> I think that it would be preferable for the audio to be stripped from the videos.

This is simply to prevent any potential delay by decoupling the feature release from video modifications.

By the way, I tried:

> {{ $youtube = printf "{{< youtube id=%s mute=true >}}" $videoID }}

But it doesn’t mute the videos. There’s no error, but the setting is not applied. (in my local env)

Reply:
Oh right, support for the arguments to the youtube shortcode was added in Hugo 0.125.0, and the site is currently only using 0.122.0.

I'll investigate upgrading Hugo later today, if only to the latest 0.125.x. WDYT?